src/Mapping/FmtMapping.php

```diff
@@ -79,6 +79,10 @@ public function __construct()
             'CodeProduit' => function() {
                 return 'COLD';
             },
+            'Poids' => function(OrderInterface $order) {
+                //@TODO get current shipment only
+                return $order->getShipments()[0]->getShippingWeight();
```

Comment:
Maybe we have a method `->first()` on the `getShipments` return ?
Are we sure we have a shipment ?

Reply:
I updated the code with first method on shipment collection.
Without that, exporting orders to coliship doesn't work. 
In a perfect world, we must pass a Shipment Instance in order to an Order Instance.